keyboards/tweetydabird/lotus58/keymaps/default/keymap.c

```diff
@@ -0,0 +1,92 @@
+	  _______,   KC_F1,     KC_F2,     KC_F3,     KC_F4,     KC_F5,     _______,      _______,    KC_F6,    KC_F7,      KC_F8,     KC_F9,    KC_F10,    _______,
+     _______,   _______,   _______,   _______,   _______,   _______,                             _______,   _______,   _______,   _______,   _______,   _______,
+     _______,   _______,   _______,   _______,   _______,   _______,                             _______,   _______,   _______,   _______,   _______,   _______,
+     _______,   _______,   _______,   _______,   _______,   _______,    _______,      _______,   _______,   _______,   _______,   _______,   _______,   _______,
+            					      _______,   _______,   _______,  _______,         _______,  _______,   _______,   _______
+),
+
+[3] = LAYOUT(
+  XXXXXXX, XXXXXXX, XXXXXXX ,  XXXXXXX, XXXXXXX, XXXXXXX, _______,     _______, XXXXXXX, XXXXXXX, XXXXXXX, RGB_MOD, RGB_SAI, RGB_TOG,
+  QK_BOOT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                       XXXXXXX, XXXXXXX, XXXXXXX, RGB_RMOD, RGB_SAD, RGB_M_P,
+  XXXXXXX, XXXXXXX, XXXXXXX,   XXXXXXX, XXXXXXX, KC_CAPS,                       XXXXXXX, XXXXXXX, XXXXXXX, RGB_HUI, RGB_VAI, RGB_M_B,
+  XXXXXXX, XXXXXXX, XXXXXXX,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, RGB_HUD, RGB_VAD, RGB_M_R,
+								_______, _______, _______, _______,     _______, _______, _______, _______ 
+  )
+};
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [1] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [2] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [3] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_VOLD, KC_VOLU) }
+};
+#endif
+
+#ifdef OLED_ENABLE
+static void print_status_narrow(void) {
+	// Create OLED content
+    oled_write_P(PSTR("\n"), false);
+	oled_write_P(PSTR(""), false);
+    oled_write_P(PSTR("Lotus -58-"), false);
+	oled_write_P(PSTR("\n"), false);
+
+    // Print current layer
+    oled_write_P(PSTR("Layer"), false);
+    switch (get_highest_layer(layer_state)) {
+        case 0:
+            oled_write_P(PSTR("-Base\n"), false);
+            break;
+        case 1:
+            oled_write_P(PSTR("-Num \n"), false);
+            break;
+        case 2:
+            oled_write_P(PSTR("-Func\n"), false);
+            break;
+        case 3:
+            oled_write_P(PSTR("-Sys \n"), false);
+            break;
+        default:
+            oled_write_P(PSTR("Undef"), false);
+    }
+	
+    oled_write_P(PSTR("\n"), false);
+    led_t led_usb_state = host_keyboard_led_state();
+    oled_write_ln_P(PSTR("Caps- lock"), led_usb_state.caps_lock);
+	
+}
+
+bool oled_task_user(void) {
+	// Render the OLED
+	if (is_keyboard_left()) {
+		print_status_narrow();
+		//render_logo();
+    } else {
+		print_status_narrow();
+        //render_logo();
+    }
```

Comment:
Could be reduced into
```suggestion
		print_status_narrow();
```